CHANGELOG.md

```diff
@@ -78,6 +78,8 @@ different versioning scheme, following the Haskell community's
 * **Breaking change** When using Bond-over-gRPC, the generated `ClientCore::Async*`
   functions now accept `std::shared_ptr<grpc::ClientContext>`
   as the last parameter instead of as the first.
+* **Breaking change** When using Bond-over-gRPC, the client callback now directly accepts
+  `bond::ext::gRPC::unary_call_result<Response>` (drops the `std::shared_ptr`).
```

Comment:
The changelog should also capture the change to `unary_call_result` to now have accessor functions instead of fields.